src/Throttlers/CacheThrottler.php

```diff
@@ -177,9 +178,17 @@ protected function hitRedis()
                'if v>1 then return v '.
                'else redis.call(\'setex\', KEYS[1], ARGV[1], 1) return 1 end';
 
-        $this->number = $this->store->connection()->eval(
-            $lua, 1, $this->store->getPrefix().$this->key, max(1, $this->time * 60));
+        $this->number = $this->store->connection()->eval($lua, 1, $this->computeRedisKey(), $this->time);
```

Comment:
- Was being defensive while using max, if someone specified negative time in config :)
- As time is in minutes, multiply by 60 was correct, I think. Lua usage setex which expects time arg in seconds. I think this is buggy change. We might need to have redis driver specific tests/some refactor.

@GrahamCampbell .